fmralign/examples/pairwise_alignment.py

```diff
@@ -0,0 +1,26 @@
+from nilearn.datasets import fetch_neurovault, neurovault, fetch_neurovault_ids
```

Comment:
OK, this looks unfinished. I think it is always good to start an example by describing your intention.